automation_tools/__init__.py

```diff
@@ -425,9 +425,24 @@ def setup_scap_client():
     run('yum -y install puppet-foreman_scap_client', warn_only=True)
 
 
-def install_discovery_image():
-    """Task to install foreman discovery image."""
+def setup_foreman_discovery():
+    """Task to setup foreman discovery."""
+
+    admin_password = os.environ.get('ADMIN_PASSWORD')
+    if admin_password is None:
+        admin_password = os.environ.get('ADMIN_PASSWORD', 'changeme')
```

Comment:
This is kinda wonky. The `get` method is already designed to handle this bit of logic:

> get(key[, default])
>
> > Return the value for key if key is in the dictionary, else default. If default is not given, it defaults to None, so that this method never raises a KeyError.

So the above three lines can be simplified to:

```python
admin_password = os.environ.get('ADMIN_PASSWORD', 'changeme')
```

Or, if `''` is an acceptable value for `ADMIN_PASSWORD`, do this instead:

try:
    admin_password = os.environ['ADMIN_PASSWORD']
except KeyError:
    admin_password = 'changeme'
```

Also, please drop the blank line between the docstring and the start of the method logic.